test/integration/stats_integration_test.cc

```diff
@@ -146,9 +146,25 @@ class ClusterMemoryTestHelper : public BaseIntegrationTest {
   ClusterMemoryTestHelper()
       : BaseIntegrationTest(testing::TestWithParam<Network::Address::IpVersion>::GetParam()) {}
 
-  static size_t computeMemory(int num_clusters) {
+  static size_t computeMemoryDelta(int initial_num_clusters, int initial_num_hosts,
```

Comment:
nit: prefer explicit unsigned types vs. int, e.g., uint32_t, but not your code and feel free to cleanup in a follow up.